### PR TITLE
Blue border around UIKitView

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**
Simplify modifiers to be compatibe with tutorial doc

### CODE REVIEW

File: examples/interop/ios-uikit-in-compose/shared/src/iosMain/kotlin/UseUITextField.kt [hunk 1]
```diff
@@ -37,7 +37,7 @@ fun UseUITextField() {
             )
             textField
         },
-        modifier = Modifier.padding(4.dp).fillMaxWidth().height(30.dp).border(2.dp, Color.Blue),
+        modifier = Modifier.fillMaxWidth().height(30.dp),
```
Comment (igordmn) [inline examples/interop/ios-uikit-in-compose/shared/src/iosMain/kotlin/UseUITextField.kt:40]:
```suggestion
        modifier = modifier.fillMaxWidth().height(30.dp),
```
We should add `modifier: Modifier = Modfier` to arguments as well

File: examples/interop/ios-uikit-in-compose/shared/src/iosMain/kotlin/main.ios.kt [hunk 2]
```diff
@@ -21,14 +22,29 @@ fun ComposeEntryPoint(): UIViewController =
...
+                UseUITextField()
+            }
         }
     }
+
+/**
+ * Border to easily understand the size of UIKitView
+ */
+@Composable
+fun BlueBorder(content: @Composable () -> Unit) {
```
Comment (igordmn) [inline examples/interop/ios-uikit-in-compose/shared/src/iosMain/kotlin/main.ios.kt:46]: The Jetpack Compose recomendation is to use modifier for this case:
```
    ComposeUIViewController {
        Column(
            Modifier
                .fillMaxSize()
                .windowInsetsPadding(WindowInsets.systemBars),
            horizontalAlignment = Alignment.CenterHorizontally
        ) {
            Text("How to use UIKitView inside Compose")

            UIKitView(
                factory = { MKMapView() },
                modifier = Modifier.padding(4.dp).border(2.dp, Color.Blue).size(300.dp),
                update = {},
            )

            Spacer(Modifier.size(20.dp))

            UseUITextField(Modifier.padding(4.dp).border(2.dp, Color.Blue))
        }
    }
```

File: examples/interop/ios-uikit-in-compose/shared/src/iosMain/kotlin/UseUITextField.kt (hunk 1 above)
Comment (dima-avdeev-jb) [reply examples/interop/ios-uikit-in-compose/shared/src/iosMain/kotlin/UseUITextField.kt]: Done

File: examples/interop/ios-uikit-in-compose/shared/src/iosMain/kotlin/main.ios.kt (hunk 2 above)
Comment (dima-avdeev-jb) [reply examples/interop/ios-uikit-in-compose/shared/src/iosMain/kotlin/main.ios.kt]: Done

**Review by igordmn — APPROVED** [review]

**Pull request merged**